modules/cnf-topology-aware-lifecycle-manager-troubleshooting.adoc

```diff
@@ -435,3 +433,13 @@ ERROR	controller-runtime.manager.controller.clustergroupupgrade	Reconciler error
 sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
 ----
 <1> Displays the error.
+
+[discrete]
```

Comment:
This troubleshooting guide addresses OCPBUGS-1372, OCPBUGS-2812 and OCPBUGS-2813

Reply:
@imiller0